logic/src/main/nl/jft/logic/match/Goal.java

```diff
@@ -1,8 +1,43 @@
 package nl.jft.logic.match;
 
+import nl.jft.common.Identifiable;
+import nl.jft.logic.participant.Participant;
+
+import java.time.LocalDateTime;
+import java.util.Objects;
+
 /**
+ * A {@code Goal} represents a {@code Goal} made by a {@link nl.jft.logic.participant.Participant} in a {@link Match}.
+ * It contains metadata about the {@code Goal} such as who scored it, and when it was scored.
+ *
  * @author Lesley
  */
-public class Goal {
+public class Goal implements Identifiable {
+
+    private final int id;
+    private final Participant participant;
+    private final LocalDateTime time;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "time" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://92.110.24.119:9000/coding_rules#rule_key=squid%3AS1068)